css/styles.css

```diff
@@ -1,6 +1,7 @@
 /* dynamic variables CSS */

```

Comment:
You missed picture for the first section

Reply:
You missed border for the team member card

Reply:
You should move border to `description` class and apply hover effect for all `example` card on second page